src/utils.ts

```diff
@@ -302,3 +308,28 @@ export const getVersion = (): string | void => {
     return packageJson.version;
   } catch {}
 };
+
+export const fileNameWithoutExtension = (fileName: string): string => {
+  return fileName.split('.').slice(0, -1).join('.');
+};
+
+export const readDirIntoShotItems = (path: string): ShotItem[] => {
```

Comment:
I don't get this function name just from reading this :D mind explaining @chriskalmar ?

Reply:
turns `someShotsName.png` into `someShotsName`

Reply:
this presumes we are having our images in `.png` format always, right?